src/Utilities/ErrorHandling/FormatStacktrace.hpp

```diff
@@ -0,0 +1,73 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+/// \file
+/// Formatter for Boost stacktrace, following these docs:
+/// https://www.boost.org/doc/libs/1_78_0/doc/html/stacktrace/getting_started.html#stacktrace.getting_started.global_control_over_stacktrace_o
+
+#pragma once
+
+#include <boost/stacktrace/stacktrace_fwd.hpp>
+
+#include <cstddef>
+#include <ostream>
+#include <string>
+
+namespace boost {
+namespace stacktrace {
+
+template <class CharT, class TraitsT, class Allocator>
+std::basic_ostream<CharT, TraitsT>& do_stream_st(
+    std::basic_ostream<CharT, TraitsT>& os,
+    const basic_stacktrace<Allocator>& backtrace) {
+  const std::streamsize standard_width = os.width();
+  const size_t frames = backtrace.size();
+  for (size_t i = 0; i < frames; ++i) {
+    const auto& frame = backtrace[i];
+    const std::string& symbol_name = frame.name();
+    const std::string& source_file = frame.source_file();
+    const size_t source_line = frame.source_line();
+    // Enumerate frame number
+    os.width(3);
+    os << i;
+    os.width(standard_width);
+    os << ". ";
+    // Skip frames originating in error handling code
+    if (symbol_name.find("abort_with_error_message") != std::string::npos or
+        source_file.find("AbortWithErrorMessage.cpp") != std::string::npos) {
+      os << "[error handling]\n";
+      continue;
+    }
+    // Print symbol name. Abbreviate if necessary to avoid filling the screen
+    // with templates.
+#ifdef SPECTRE_ABBREV_BACKTRACE_SYMBOLS
```

Comment:
Can we base this on an environment variable instead of compile-time?  Seems like a pain to have to recompile if you need a verbose backtrace.

Reply:
I pushed a fixup with.. something. Please take a look :)